src/test/java/de/retest/web/PageFrameIT.java

```diff
@@ -31,6 +31,8 @@ void page_frame_html_should_be_checked( final WebDriver driver ) throws Exceptio
 		final Path simplePagePath = Paths.get( "src/test/resources/pages/page-frame.html" );
 		driver.get( simplePagePath.toUri().toURL().toString() );
 
+		Thread.sleep( 1000 );
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this use of "Thread.sleep()". ([squid:S2925](https://sonarcloud.io/organizations/retest/rules#rule_key=squid%3AS2925))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.retest%3Arecheck-web&pullRequest=84&issues=AWahP8EYrChX7_AlIBKD&open=AWahP8EYrChX7_AlIBKD)

[ISSUE_KEY]: AWahP8EYrChX7_AlIBKD